docs/source/guides/orchestrator_transcoder_gpu.rst

```diff
@@ -0,0 +1,123 @@
+
+Run the orchestrator
+********************
+
+The following flags are required to run an orchestrator with GPU transcoding: 
+
+::
+
+    livepeer -orchestrator \
+        -transcoder \
+        -network mainnet \
+        -ethURL <ETH RPC URL> \
+        -nvidia <NVIDIA GPU DEVICE IDs>
+
+You can use all GPUs accessible on your machine by specifying all GPU IDs in the argument passed to the :code:`-nvidia` flag.
+For example, the following command will use GPUs 0-5 when transcoding:
+
+::
+
+    livepeer -orchestrator \
+        -transcoder \
+        -network mainnet \
+        -ethURL http://127.0.0.1:8545 \
+        -nvidia 0,1,2,3,4
+
+You can use a subset of the GPUs accessible on your machine by excluding GPU IDs in the argument passed to the :code:`-nvidia` flag.
+For example, the following command will not use GPU #1 when transcoding:
+
+::
+
+    livepeer -orchestrator \
+        -transcoder \
+        -network mainnet \
+        -ethURL http://127.0.0.1:8545 \
+        -nvidia 0,2,3,4
+
+You can use an optional :code:`-v 6` flag to set the logging level to verbose in order to observe more detailed transcoding logs:
+
+::
+
+    livepeer -orchestrator \
+        -transcoder \
+        -network mainnet \
+        -ethURL http://127.0.0.1:8545 \
+        -nvidia 0,1,2,3,4 \
+        -v 6
+
+On startup, the orchestrator will automatically run a test to confirm that it is able to transcode using the specified GPUs.
+The orchestrator will exit if this test fails. If the test passes, you should see the following message in the log output without any
+additional error messages following it indicating that your orchestrator is ready to transcode streams using the specified GPUs:
+
+::
+
+    Received Ping request  
+
+Transcoding information
+***********************
+
+If you set the logging level to verbose via the :code:`-v 6` flag, then you should see messages similar to the following when your orchestrator is transcoding:
+
+::
+
+    Downloaded segment manifestID=e1353c56 sessionID=29b089af seqNo=0 dur=65.220433ms
+    LB: Creating transcode session for e1353c56
+    LB: Created transcode session for e1353c56_1
+    LB: Transcode submitted for e1353c56_1
+    Transcoding of segment manifestID=e1353c56 sessionID=29b089af seqNo=0 took=396.34309ms
+
+You can observe the download speed (from the broadcaster to the orchestrator) in the :code:`dur` field of the following line:
+
+::
+ 
+    Downloaded segment manifestID=e1353c56 sessionID=29b089af seqNo=0 dur=65.220433ms
+
+You can observe the transcoding speed specified in the :code:`took` field of the following line:
+
+::
+
+    Transcoding of segment manifestID=e1353c56 sessionID=29b089af seqNo=0 took=396.34309ms
+
+In the above line, the segment was transcoded in ~396.34ms.
+
+You can also verify that the GPUs are being used for transcoding as well as the video encoder and decoder utilization on the GPUs using :code:`nvidia-smi`:
```

Comment:
It seems to me that transcoding would always use the encoder and decoder chips ? In that case WDYT about slightly rewording this to 

You can verify that the GPUs are being used for transcoding by checking the video encoder (:code:`enc`) and decoder (:code:`dec`) utilization on the GPUs using :code:`nvidia-smi dmon`:

Reply:
Fixed in 5201ba0